docs/reference/operator_parameters.md

```diff
@@ -212,6 +212,9 @@ words.
   pods. Only used when combined with
   [kube2iam](https://github.com/jtblin/kube2iam) project on AWS. The default is empty.
 
+* **aws_region**
```

Comment:
this should be in the "AWS or GSC interaction" section of the docs.

Reply:
it already is :)

Reply:
Missed that, sorry.